sdks/router-sdk/src/utils/getOutputAmount.ts

```diff
@@ -0,0 +1,20 @@
+import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
+import { Pool as V4Pool } from '@uniswap/v4-sdk'
+import { Pair } from '@uniswap/v2-sdk'
+import { Pool as V3Pool } from '@uniswap/v3-sdk'
+
+type TPool = Pair | V3Pool | V4Pool
+
+export async function getOutputAmount(
+  pool: TPool,
+  amountIn: CurrencyAmount<Currency>
+): Promise<[CurrencyAmount<Currency>, TPool]> {
+  if (pool instanceof V4Pool) {
+    if (pool.involvesCurrency(amountIn.currency)) {
+      return await pool.getOutputAmount(amountIn)
+    } else if (pool.involvesCurrency(amountIn.currency.wrapped)) {
```

Comment:
I think this else if branch is redundant since the default behavior of this function is to return the outputAmount given the wrapped amountIn

Reply:
ahh good point, I had a thrown error in there if it was neither, but realized it would never get hit and took it out.